VERSION

```diff
@@ -100,7 +100,7 @@ libompitrace_so_version=40:0:0
 # components-don't-affect-the-build-system abstraction.
 
 # OMPI layer
-libmca_ompi_common_ompio_so_version=40:0:0
+libmca_ompi_common_ompio_so_version=41:0:0
```

Comment:
are you sure you don't want to change the age here as well?

Reply:
I don't think so?  As I mentioned in the commit log, there was a non-backward compatible change in the interface into the library.

Reply:
I think @hppritcha is correct - here are the libtool rules:
```
# - If the library source code has changed at all since the last
#   update, then increment revision (`c:r:a' becomes `c:r+1:a').
#
# - If any interfaces have been added, removed, or changed since
#   the last update, increment current, and set revision to 0.
#
# - If any interfaces have been added since the last public release,
#   then increment age.
#
# - If any interfaces have been removed since the last public release,
#   then set age to 0.
```
So an interface was added, but nothing removed - which means you increment the first number, set the second to 0, and increment the third.

Reply:
An interface was "changed", not "added".  See mca_common_ompio_build_io_array() in https://github.com/open-mpi/ompi/commit/0e0d45f733544476cdd22c4a1a9d765ed2df42a9#diff-498be05eb92cb6fc482cd0125a5f2021.  It added a new parameter, which is an interface change.  So I incremented current, set revision to 0, and didn't change age (which was already 0).

Reply:
Sorry, I thought I saw somewhere above that it was added - yes, your version looks correct to me.